components/klaudsolcms/modals/AddEditAnotherFieldModal.js

```diff
@@ -30,7 +30,7 @@ export default function AddEditAnotherFieldModal({formParams, show, onClose, onC
         <tbody>
             <tr>
               <td><Field name='name' className='input_text' /></td>
-              <td>
+              <td style={{width: '150px'}}>
```

Comment:
Let's avoid using inline styles. Kindly add a classname and adjust the width via css.